rust/operator-binary/src/airflow_db_controller.rs

```diff
@@ -202,6 +204,8 @@ pub async fn reconcile_airflow_db(airflow_db: Arc<AirflowDB>, ctx: Arc<Ctx>) ->
                         )
                         .await
                         .context(ApplyStatusSnafu)?;
+                } else {
+                    return SecretNotFoundSnafu.fail();
```

Comment:
There is a `client.get_opt(secret...)` call. I would not use `get_opt` but just `get` and error out there (not tested)?

Reply:
done